src/main/java/de/tum/in/www1/artemis/service/BuildLogEntryService.java

```diff
@@ -43,6 +43,9 @@ public class BuildLogEntryService {
     @Value("${artemis.continuous-integration.build-log.file-expiry-days:30}")
     private int expiryDays;
 
+    @Value("${artemis.build-logs-path:./build-logs}")
+    private String buildLogsPath;
```

Comment:
```suggestion
    private Path buildLogsPath;
```
Can Spring automatically convert it to path? `ProgrammingExerciseExportService` seems to do it like that.